Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Description: relax serde_json dep from 1.0.81 to 1.0.80

Upstream indicated no reason to need serde_json 1.0.81 specifically
and uploading serde_json > 1.0.80 to debian runs into #1015743 so it
is simpler to just relax this dependency here for now.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -68,7 +68,7 @@
 features = ["derive"]
 
 [dependencies.serde_json]
-version = "1.0.81"
+version = "1.0.80"
 
 [dependencies.tempfile]
 version = "3.1"
